app/src/main/res/layout-v26/fragment_detail.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
proc je to ve slozce layout-v26? to znamena, ze se to zobrazi jen na verzich 26+

Reply:
zkusila jsem ho presunout pryc, uvidime